app/components/agenda/modal.js

```diff
@@ -11,30 +11,55 @@ export default class AgendaModalComponent extends Component {
 
   @tracked currentlyEditing;
 
+  isNew = false;
+  get afterSave() {
```

Comment:
I prefer an empty line between class fields and methods, makes it a bit more readable (but again nitpicking)